require/resolve.go

```diff
@@ -274,3 +283,9 @@ func isFileOrDirectoryPath(path string) bool {
 
 	return result
 }
+
+func moduleDebug(msg string) {
+	if os.Getenv("NODE_DEBUG") == "module" {
```

Comment:
os.Getenv will do a syscall every time. Again, not the most efficient way. It should be cached once and then re-used. I think node works this way.

Reply:
Also, NODE_DEBUG may contain comma-separated values.